lib/phoenix/tracker.ex

```diff
@@ -110,8 +117,9 @@ defmodule Phoenix.Tracker do
   @doc """
   Tracks a presence.
 
-    * `server_name` - The registered name of the tracker server
-    * `pid` - The Pid to track
+    * `server` - The pid or registered name of the tracker server
+    * `pid` - The pid to track. The tracker will monitor the process and
+      and automatically untrack it when it exits.
```

Comment:
I wasn't 100% sure on this, but decided to document my understanding of what `pid` is used for.

Actually, from the code, it looks like the processes are actually linked instead of monitored which is surprising.  The tracker server has called `Process.flag(:trap_exit, true)` so that exits or crashes from linked tracked processes are converted into messages instead of taking down the tracker (which is good) but I believe a crash of the tracker will take down a tracked process due to the bi-directional nature of the link.  That seems dangerous.  Do we really want tracked processes to be forced to exit if/when the tracker crashes?  Can the tracker server use monitoring instead of a link?